conseil-common/src/main/scala/tech/cryptonomic/conseil/common/bitcoin/BitcoinPersistence.scala

```diff
@@ -0,0 +1,143 @@
+package tech.cryptonomic.conseil.common.bitcoin
+
+import java.sql.Timestamp
+import java.time.Instant
+
+import cats.Id
+import cats.effect.{Concurrent, Resource}
+import com.typesafe.scalalogging.LazyLogging
+import slick.jdbc.PostgresProfile.api._
+
+import tech.cryptonomic.conseil.common.util.Conversion
+import tech.cryptonomic.conseil.common.util.Conversion.Syntax._
+import tech.cryptonomic.conseil.common.bitcoin.rpc.json.{Block, Transaction, TransactionInput, TransactionOutput}
+import tech.cryptonomic.conseil.common.bitcoin.BitcoinPersistence._
+
+
+/**
+  * Bitcoin persistence into the database using Slick.
+  */
+class BitcoinPersistence[F[_]: Concurrent] extends LazyLogging {
+
+  /**
+    * Create [[DBIO]] seq with block (with transactions, inputs and outputs) that can be wrap into one transaction.
+    *
+    * @param block JSON_RPC block
+    * @param transactions JSON_RPC block's transactions
+    */
+  def createBlock(
```

Comment:
Can it be renamed to `writeBlock`/`insertBlock` for consistency with other existing persistency classes?